modules/mux/targets/host/test/UnitCL/host_clGetDeviceInfo.cpp

```diff
@@ -55,6 +55,10 @@ TEST_F(host_clGetDeviceInfoTest, Name) {
   const char *arch = "x86";
 #elif defined(__x86_64__) || defined(_M_X64)
   const char *arch = "x86_64";
+#elif defined(__riscv) && __riscv_xlen == 64
+  const char *arch = "riscv64";
+#elif defined(__riscv__)
```

Comment:
Why the difference between `__riscv` and `__riscv__`? Is that really how they're defined?

Reply:
Yeah this doesn't look right. As I said above, it *should* be possible to check `__riscv_xlen` against 32 (carefully, mind)

Reply:
https://github.com/riscv-non-isa/riscv-toolchain-conventions claims "__riscv: defined for any RISC-V target. Older versions of the GCC toolchain defined __riscv__." but when I look through GCC's history, it never had `__riscv__`, it only had `__riscv` ever since it was first supported as a target (https://github.com/gcc-mirror/gcc/commit/09cae7507d9e88f2b05cf3a9404bf181e65ccbac#diff-6427e55c1bb73417ca77ff9c33ade2fe85a3796eb99c7de329c67717d45fb27d, look for `riscv_cpu_cpp_builtins`). This was GCC 7 and we already require newer GCC than GCC 7. I think we can simply get rid of `__riscv__` entirely.